src/field/captcha/recaptcha.jsx

```diff
@@ -59,7 +83,10 @@ export class ReCAPTCHA extends React.Component {
   componentDidMount() {
     ReCAPTCHA.loadScript(this.props, document.body, (err, scriptNode) => {
       this.scriptNode = scriptNode;
-      this.widgetId = window.grecaptcha.render(this.ref.current, {
+      const global = globalForProvider(this.props.provider);
+
+      // if this is enterprise then we change this to window.grecaptcha.enterprise.render
+      this.widgetId = global.render(this.ref.current, {
```

Comment:
Let's rename this, `global` is a little too ambiguous and nondescript for me. How about `provider`, or `recaptchaProvider`?